lib/time/time.go

```diff
@@ -99,6 +100,16 @@ func fromTimestamp(thread *starlark.Thread, _ *starlark.Builtin, args starlark.T
 	return Time(time.Unix(x, 0)), nil
 }
 
+// Converts the given timestamp corresponding to the total amount of nanoseconds
```

Comment:
A timestamp (aka UNIX time_t) is defined as the number of seconds since Jan 1, 1970, which may be an integer, or a fixed-point number (pair of int64) if sub-second precision is needed. It would be confusing for this function to introduce the new concept of a "timestamp" that is a quantity of nanoseconds.

Instead, change the existing from_timestamp function so that it accepts either one int (a number of seconds) or two ints (sec and nsec). It could additionally accept one float, a number of seconds. There is no need for a new function.

Also, the functions provided by this module should be documented at `Module`. I realize this is a preexisting problem, but would you mind addressing it while you're working here? (Look at json and math for examples.)

Reply:
@adonovan I understand your point of view even if it is not really helpful in my use case since I have timestamps expressed in nanoseconds so I will need to explicitly split seconds from nanoseconds which is not really convenient. Anyway I addressed your remarks.